### PR TITLE
PHPStan 1.0 and Psalm level 2

### PR DESCRIPTION
**Pull request by jordisala1991**
Label as pedantic, since it does not modify any relevant code on src.

### CODE REVIEW

File: tests/App/AppKernel.php [hunk 1]
```diff
@@ -33,6 +33,11 @@
 use Symfony\Component\Routing\RouteCollectionBuilder;
 use Symfony\Component\Security\Http\Authentication\AuthenticatorManager;
 
+/**
+ * @psalm-suppress PropertyNotSetInConstructor
+ *
+ * @see https://github.com/psalm/psalm-plugin-symfony/pull/220
```
Comment (VincentLanglet) [inline tests/App/AppKernel.php:39]: I've created https://github.com/psalm/psalm-plugin-symfony/issues/225

**Comment by jordisala1991** [comment]
Will need: https://github.com/sonata-project/dev-kit/pull/1751

File: tests/App/AppKernel.php (hunk 1 above)
Comment (jordisala1991) [reply tests/App/AppKernel.php:39]: Let's see if it gets solved, the PR I linked it has the problem described as the last message too.

**Review by core23 — APPROVED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Comment by VincentLanglet** [comment]
Thanks

**Pull request merged**